api/lib/interceptors.ts

```diff
@@ -1,43 +1,43 @@
-import * as exceptionFormatter from 'exception-formatter'
-import * as log4js from 'log4js'
-import { config } from '../../config'
-import { shorten, valueOrNull } from '../lib/util'
+import * as exceptionFormatter from 'exception-formatter';
+import * as log4js from 'log4js';
+import { config } from '../../config';
+import { shorten, valueOrNull } from '../lib/util';
 
 const exceptionOptions = {
     maxLines: 1,
```

Comment:
question: is the comma needed, is it best practice

Reply:
it is neither, some people prefer it though. I generally don't leave trailing commas